src/website/views/templates/autopay.pug

```diff
@@ -0,0 +1,592 @@
+                a.m-dropdown__menu-item(href='#creditDebitCardBase', name='Credit') Credit or debit card
+      .a-card#checkingAccountBase.-mt--1
+        .a-card__header.-py--0.-bg--grey-10
+          h6 Add a checking account
+        .a-card__content.-p--3
+          fieldset
+            .a-grid.-noGutter
+              div.a-col.-w--12.-w-md--6
+                .m-formGroup.-labeled.-mb--2.-mr-md--1
+                  label.a-label(for='checking-routing-number') Routing number
+                  input(type="text", class="a-input -xl", id='checking-routing-number', placeholder="")
+              div.a-col.-w--12.-w-md--6
+                .m-formGroup.-labeled.-mb--2
+                  label.a-label(for='checking-account-number') Account number
+                  input(type="text", class="a-input -xl", id='checking-account-number', placeholder="")
+              div.a-col.-w--12
+                .-d--flex.-justify-content--center.-mt--2
+                  img.-mw--100.-h--auto(src='../../assets/images/autopay-check-hint.png', alt="savings account")
+      .a-card#savingsAccountBase.-mt--1.-d--none
+        .a-card__header.-py--0.-bg--grey-10
+          h6 Add a savings account
+        .a-card__content.-p--3
+          fieldset
+            .a-grid.-noGutter
+              div.a-col.-w--12.-w-md--6
+                .m-formGroup.-labeled.-mb--2.-mr-md--1
+                  label.a-label(for='savings-routing-number') Routing number
+                  input(type="text", class="a-input -xl", id='savings-routing-number', placeholder="")
+              div.a-col.-w--12.-w-md--6
+                .m-formGroup.-labeled.-mb--2
+                  label.a-label(for='savings-account-number') Account number
+                  input(type="text", class="a-input -xl", id='savings-account-number', placeholder="")
+              div.a-col.-w--12
+                .-d--flex.-justify-content--center.-mt--2
+                  img.-mw--100.-h--auto(src='../../assets/images/autopay-check-hint.png', alt="savings account")
+      .a-card#creditDebitCardBase.-mt--1.-d--none
+        .a-card__header.-py--0.-bg--grey-10
+          h6 Add a credit or debit card
+        .a-card__content.-p--3
+          fieldset
+            .a-grid.-noGutter
+              div.a-col.-w--12
+                .m-formGroup.-labeled.-mb--2
+                  label.a-label(for='name') Name on card
+                  input(type="text", class="a-input -xl", id='name', placeholder="")
+              div.a-col.-w--12
+                .m-formGroup.-labeled.-mb--2
+                  div.-d--flex.-justify-content--between.-align-items--center
+                    label.a-label(for='number') Card number
+                    img(src='../../assets/images/autopay-credit-cards.jpg', alt="credit cards")
+                  input(type="text", class="a-input -xl", id='number', placeholder="")
+              div.a-col.-w-md--4.-w-sm--12
+                .m-formGroup.-labeled
```

Comment:
Looks like we'll need -mb--2 for small displays to provide sufficient margin between expiration and zip code 

![image](https://user-images.githubusercontent.com/130654/56101027-abce1c00-5ed4-11e9-984a-b40cc75f0aba.png)

Reply:
done